packages/zowe-explorer/src/trees/dataset/DatasetUtils.ts

```diff
@@ -67,45 +67,45 @@ export class DatasetUtils {
     }
 
     /**
-     * Get the language ID of a Data Set for use with `vscode.languages.setTextDocumentLanguage`
+     * Get the file extension for a Data Set (or data set member) based on its name or its PDS name.
      */
-    public static getLanguageId(label: string): string | null {
+    public static getExtension(label: string): string | null {
```

Comment:
Curious if we'd want to define a mapping of extensions to ds qualifiers as a constant? For example:
```javascript
const EXT_MAPPING = {
  ".c": ["C"],
  ".jcl": ["JCL", "JCLLIB", "CNTL", "PROC", "PROCLIB"]
}
```

This could simplify the code below, as well as in `extensionRemovedFromPath` which could call `Object.keys` to iterate over all the extensions.

Reply:
I had restored the previous implementation for this fix, but I think it makes sense to clean it up. I like this idea, but I used a map instead so I can get the extension & array of matches using `Map.entries`.

Addressed in d7a30e4